<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YandexWeatherDayWidget</class>
 <widget class="QWidget" name="YandexWeatherDayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>172</width>
    <height>170</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:transparent;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="weekDayLabel">
     <property name="styleSheet">
      <string notr="true">QLabel { background-color : rgba(230,230,240,180); color : black; }</string>
     </property>
     <property name="text">
      <string>WeekDay</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dateLabel">
     <property name="styleSheet">
      <string notr="true">QLabel { background-color : rgba(230,230,240,180); color : black; }</string>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="weatherTypeWidget">
     <property name="styleSheet">
      <string notr="true">QLabel { background-color : rgba(200,200,170,180); color : black; }</string>
     </property>
     <property name="text">
      <string>WeatherType</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dayTemperatureLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background-color : rgba(200,200,170,180); color : black; }</string>
     </property>
     <property name="text">
      <string>DayTemp</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nightTemperatureLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background-color : rgba(200,200,170,180); color : black; }</string>
     </property>
     <property name="text">
      <string>NightTemp</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
